src/components/History/History.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Card from '../Card/Card';
+import { CleanedArticle } from '../../Models';
+import '../Feed/Feed.css'
+
+interface HistoryProps {
+  history: CleanedArticle[] | any
+  updateHistory: (localHistory: string[]) => void;
+}
+
+const History = ({ history, updateHistory }: HistoryProps): JSX.Element => {
+
+  const articleCards = history.map((article: CleanedArticle) => {
```

Comment:
I don't think this map works because history is an array of ids. Could you pass the articles and filter using history so you can map over those filtered articles?